tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/WafNative.cs

```diff
@@ -99,32 +99,34 @@ internal WafNative(IntPtr handle)
 
         private delegate IntPtr ObjectInvalidDelegate(IntPtr emptyObjPtr);
 
-        private delegate IntPtr ObjectStringLengthDelegateX64(IntPtr emptyObjPtr, string s, ulong length);
+        private delegate IntPtr ObjectStringLengthDelegate(IntPtr emptyObjPtr, string s, ulong length);
 
-        private delegate IntPtr ObjectStringLengthDelegateX86(IntPtr emptyObjPtr, string s, uint length);
+        private delegate DdwafObjectStruct ObjectStringLengthDelegateX86(IntPtr emptyObjPtr, string s, uint length);
```

Comment:
Don't think this is used anymore, because the return type looks wrong.

Reply:
Yes good catch, fixed in 50c1359ab57b5cc2a9503358084816b10d950f65